test/passing/attributes.ml

```diff
@@ -216,3 +216,11 @@ let _ = (0 + 0) [@a]
 let _ = (a.x <- 1) [@a]
 
 let _ = f ((a.x <- 1) [@a])
+
+let _ = (a.(b)) [@a]
```

Comment:
Parentheses are not necessary here. This can be fixed by modifying `Ast.parenze_exp` and calling `Indexing_op.parse`. (this is not ideal because we call parse again and again but improving this would be a lot of work)

Reply:
Implemented an alternative solution in https://github.com/ocaml-ppx/ocamlformat/pull/1300/commits/2e7280aa1726b0a7b7451de1f02412098f559f46.

Reply:
That works !